lib/src/components/day_view_components.dart

```diff
@@ -182,3 +183,73 @@ class DefaultTimeLineMark extends StatelessWidget {
     );
   }
 }
+
+/// This class is defined default view of full day event
+class FullDayEventView<T> extends StatelessWidget {
+  const FullDayEventView({
+    Key? key,
+    this.boxConstraints = const BoxConstraints(maxHeight: 100),
+    required this.events,
+    this.padding,
+    this.itemView,
+    this.titleStyle,
+    this.onEventTap,
+    required this.date,
+  }) : super(key: key);
+
+  /// Constraints for view
+  final BoxConstraints boxConstraints;
+
+  /// Define List of Event to display
+  final List<CalendarEventData<T>> events;
+
+  /// Define Padding of view
+  final EdgeInsets? padding;
+
+  /// Define custom Item view of Event.
+  final Widget Function(CalendarEventData<T>? event)? itemView;
+
+  /// Style for title
+  final TextStyle? titleStyle;
+
+  /// Called when user taps on event tile.
+  final TileTapCallback? onEventTap;
+
+  /// Defines date for which events will be displayed.
+  final DateTime date;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: boxConstraints,
+      padding: padding,
+      child: ListView.builder(
+        itemCount: events.length,
+        shrinkWrap: true,
+        itemBuilder: (context, index) => GestureDetector(
```

Comment:
Suggestion: you can use Inkwell instead GestureDetector because it reflects splash colour